proto/process/agent.proto

```diff
@@ -175,6 +175,7 @@ message Process {
 
 	uint32 key = 1;
 	int32 pid = 2;
+	int32 nsPid = 20;
```

Comment:
If this value is zero can we assume that this is from an agent with an older version? Will nsPid = pid if there is no PID name-spacing for newer agent versions?

Reply:
For the moment, we initialize it with `0` and if there's no NsPid field on the `status` file, we'll keep it as 0.

https://github.com/DataDog/gopsutil/pull/27/files#diff-733db3e76b1d1efd4e6e76f3ba740a63R108

For newer agent aversions, if there's no namespaced PID, nsPID will effectively match the process PID.